op-batcher/batcher/channel_manager.go

```diff
@@ -348,19 +354,39 @@ func (s *channelManager) Close() error {
 	}
 
 	s.closed = true
+	s.log.Info("Channel manager is closing")
 
 	// Any pending state can be proactively cleared if there are no submitted transactions
 	for _, ch := range s.channelQueue {
 		if ch.NoneSubmitted() {
+			s.log.Info("Channel has no past or pending submission and might as well not exist, channel will be dropped", "channel", ch.ID(), "")
```

Comment:
nit: This log message makes it sounds like an error or at least unusual condition. If it's expected then we can probably reduce the potential for confusion by making it say something like `Channel has no past or pending submission - dropping`.